template/bash_profile.template.bash

```diff
@@ -31,6 +31,11 @@ export SCM_CHECK=true
 # Will otherwise fall back on $HOSTNAME.
 #export SHORT_HOSTNAME=$(hostname -s)
 
+# Set Xterm/screen/Tmux title with only a short username.
+# Uncomment this (or set SHORT_USER to something else),
+# Will otherwise fall back on $USER.
+#export SHORT_USER=$(echo $USER | sed -r 's/^(.{3}).{3,}(.{3})$/\1..\2/')
```

Comment:
I did not know about `-r`, but either way I don't like the idea of moving the short username definition to the plugin, as it would make it somewhat harder to customize. Maybe truncating by default isn't a bad compromise? It's going to be disabled by default after all.